deploy/concierge/rbac.yaml

```diff
@@ -43,6 +43,10 @@ rules:
       - #@ pinnipedDevAPIGroupWithPrefix("authentication.concierge")
     resources: [ jwtauthenticators, webhookauthenticators ]
     verbs: [ get, list, watch ]
+  - apiGroups:
```

Comment:
Copyright year is not updated in this file.

Reply:
Ah, before I added the hook. 👍